docs/source/quickstart.rst

```diff
@@ -689,15 +732,64 @@ That's it! We just created a simple, read-only, ``Weather`` service.
 1. Try adding a "create" lifecycle operation to ``City``.
 2. Try adding a "delete" lifecycle operation to ``City``.
 3. Try adding :ref:`HTTP binding traits <http-traits>` to the API.
+4. Try adding :ref:`tags <tags-trait>` to shapes and filtering them out with
+   :ref:`excludeShapesByTag <excludeShapesByTag-transform>`.
 
 There's plenty more to explore in Smithy. The
 :ref:`Smithy specification <specification>` can teach you everything you need
-to know about Smithy.
-
+to know about Smithy models. :ref:`Building Smithy Models <building-models>`
+can teach you more about the build process, including how to use
+transformations, projections, plugins, and more. For more sample build
+configurations, see the `examples directory`_ of the Smithy Gradle plugin
+repository.
 
 Complete example
 ================
 
+If you followed all the steps in this guide, you should have three files, laid
+out like so::
+
+    .
+    ├── build.gradle.kts
+    ├── model
+    │   └── weather.smithy
+    └── smithy-build.json
+
+The ``smithy-build.json`` should have the following contents:
+
+.. code-block:: json
+
+    {
+        "version": "1.0"
+    }
+
+The ``build.gradle.kts`` should have the following contents:
+
+.. code-block:: kotlin
+
+    plugins {
+        id("software.amazon.smithy").version("0.5.1")
+    }
+
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+
+    dependencies {
+        implementation("software.amazon.smithy:smithy-model:[1.0, 2.0[")
```

Comment:
I'd replace `[1.0, 2.0[` with `__smithy_version__` to make this always show the latest version.